grape-roar.gemspec

```diff
@@ -17,5 +17,6 @@ Gem::Specification.new do |gem|
   gem.version       = Grape::Roar::VERSION
 
   gem.add_dependency 'grape'
-  gem.add_dependency 'roar', '>= 1.0'
+  gem.add_dependency 'roar', '~> 1.1.0'
+  gem.required_ruby_version = '>= 2.2.2'
```

Comment:
Is this actually true? I don't think Ruby 2.2.2 is required, it's just that in this project we're having hard time `bundle install`ing compatible gems. Unless you know a truly blocking reason why this gem won't work on < 2.2.2 I'd like this to be removed.